docs/CHANGELOG STYLE GUIDE.md

```diff
@@ -0,0 +1,145 @@
+
+*This style guide is partly inspired from [common-changelog](https://github.com/vweevers/common-changelog)*
+
+First, these entries are meant for our consumers from all levels so we need to keep them accessible enough for them. To make it easy on our side to achieve this, we’ve created this style guide for creating a new Changelog on a new version release.
+## General Format
+
+### 1. Version Header:
+- Format: `## [version-number]`
+- Example: `## [0.11.0]`
+  This header indicate which version the log entry description concern and follows [Semantic Versioning](https://semver.org/), it’s also sorted from the newest to latest semantic version.
+### 2. Date:
+- Format: `_YYYY-MM-DD_`
+- Example: `_2024-06-24_`
+  The date help to quickly get which version was available when and get the release timeline.
+### 3. Section Headers:
+Sections represent a group of changes scoped to an artefact or section.
+For now we restrict them to the following one to have consistency and avoid ambiguity:
+-  `### Spark`
+-  `### Catalog App`
+-  `### Icons`
+-  `### CI`
+### 4. Subsections:
+- Use subsections for specific features or changes within a component.
+- Example: `#### 🆕 New Features`
+
+### 5. Change Types:
+Sort content by importance.
+Skip content that isn't important.
+Link each change to further information.
+
+> [!NOTE]
+> You can inspire your self from [gitmoji ](https://gitmoji.dev/)
+
+Use emojis and clear keywords to categorize changes:
+- 🆕 New Feature: `🆕`
+- 🎨 Improvement: `🎨`
+- 💄 UI Change: `💄`
+- 🐛 Bug Fix: `🐛`
+- 🔧 Maintenance: `🔧`
+- 🚀 Performance Improvement: `🚀`
+- 📝 Documentation: `📝`
+- ⬆️ Dependency Update: `⬆️`
+- 🗑️ Deprecation/Removal: `🗑️`
+
+---
+## Detailed Entries
+
+### 1. Feature/Change Description:
+Provide a brief and clear description of the change.
+You can use the convention commit from the base but you’ll have to reformat it into a natural language (in the imperative mood) because it fits all contexts, including explaining changes to a stakeholder in person.
+
+You can add reference to a commit, issue or pull request at the end of the same line of a change.
+- Example: `🆕 Chips can now be selectable and closed` (#192)
+### 2.  Notice, Caution and Warning Notes:
+A release might have a separate upgrade guide or blog post that is considered essential reading.
+
+For these purposes a _notice_ must be used. This is a single-sentence paragraph with otherwise arbitrary Markdown content. Adding Markdown emphasis markers is recommended. For example (links omitted):
+```markdown
+## [2.0.0]
+
+_2024-06-21_
+
+_If you are upgrading: please see [`UPGRADING.md`](UPGRADING.md)._
+
+### Spark
+
+- **Breaking:** `Progressbar` has been moved to another package (`01e3a64`)
+```
+
+Use `> [!CAUTION]` and `> [!WARNING]` for important notices that may impact users like breaking changes, being from compilation or ui.
+- Example:
+```markdown
+> [!CAUTION] > The `Filled` style has been removed and may break your build if used.
+> 
```

Comment:
This example will not work on github.com
```suggestion
> [!CAUTION]
> The `Filled` style has been removed and may break your build if used.
```

Reply:
Done in 6730295